src/modules/launcher/Plugins/Microsoft.Plugin.Uri.UnitTests/UriHelper/ExtendedUriParserTests.cs

```diff
@@ -50,14 +50,25 @@ public class ExtendedUriParserTests
         [DataRow("http://[::]", true, "http://[::]/", true)]
         [DataRow("[2001:0DB8::1]", true, "https://[2001:db8::1]/", true)]
         [DataRow("http://[2001:0DB8::1]", true, "http://[2001:db8::1]/", true)]
-        [DataRow("[2001:0DB8::1]:80", true, "https://[2001:db8::1]/", true)]
+        [DataRow("[2001:0DB8::1]:80", true, "https://[2001:db8::1]:80/", true)]
         [DataRow("http://[2001:0DB8::1]:80", true, "http://[2001:db8::1]/", true)]
         [DataRow("mailto:example@mail.com", true, "mailto:example@mail.com", false)]
-        [DataRow("tel:411", true, "tel:411", false)]
         [DataRow("ftp://example.com", true, "ftp://example.com/", false)]
 
-        // This has been parsed as an application URI. Linked issue: #14260
-        [DataRow("example.com:443", true, "example.com:443", false)]
+        // Case where `domain:port`, as specified in issue #14260
+        [DataRow("tel:411", true, "https://tel:411/", true)]
```

Comment:
What about the telephone protocol handlers? Can this be a problem? Should show to show two results on this special case?

Reply:
Good point, maybe a special handler case for tel? We could keep stacking handler cases or restrict it to domain names with at least one dot

Reply:
> Good point, maybe a special handler case for tel? We could keep stacking handler cases or restrict it to domain names with at least one dot

Thought about this, this will break `localhost:80` and other cases that only have a single label. Is the best solution to let all non-web URI have two results?

Reply:
> > Good point, maybe a special handler case for tel? We could keep stacking handler cases or restrict it to domain names with at least one dot
> 
> Thought about this, this will break `localhost:80` and other cases that only have a single label. Is the best solution to let all non-web URI have two results?

Thought the same. Let's try. But I would restrict this to the one that are `host:port` and not for uris like `ms-settings:update`.

Reply:
Additionally, what is the expected behaviour of `[2001:0DB8::1]:80`? Shouldn't it be `http://[2001:0DB8::1]` instead of `https://[2001:0DB8::1]`

Reply:
> Additionally, what is the expected behaviour of `[2001:0DB8::1]:80`? Shouldn't it be `http://[2001:0DB8::1]` instead of `https://[2001:0DB8::1]`

Don't we have always https? I think the same as for ipv4.

Reply:
![image](https://user-images.githubusercontent.com/6839340/183418086-34875e2e-8f35-495b-aaa0-9d5f48a01b7a.png)
![image](https://user-images.githubusercontent.com/6839340/183418236-8482f35a-c775-47b2-b6d1-dc0908dc5e42.png)
On PT v0.61.1, this may be a separate issue

Reply:
Let's fix the ipv6/ipv4 http(s) thing in a separate step/issue/pr.

Reply:
I've verified with test cases that this has been fixed in the latest code, so I'll mark this as resolved